package.json

```diff
@@ -62,6 +62,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "test:coverage": "react-scripts test --coverage",
+    "format": "prettier-eslint \"src/**/*.js*\" --trailing-comma all --eslint-config-path ./.eslintrc --write",
```

Comment:
I kinda liked the pre-commit hook so you don’t have to think about it

Reply:
Then I'll readd it. I can run with `HUSKY_SKIP_HOOKS = 1` to avoid the hook apparently, so set up for the safe way and let the annoying people (e.g. me) deal with an env flag and the shame when their PR isn't linted correctly.